inc/datacollector/class-book.php

```diff
@@ -298,6 +320,23 @@ public function copyBookMetaIntoSiteTable( $book_id ) {
 		// pb_book_license
 		update_site_meta( $book_id, self::LICENSE, $metadata['pb_book_license'] ?? 'all-rights-reserved' );
 
+		$licensing = new Licensing;
+		$supported_types = $licensing->getSupportedTypes();
+
+		// pb_license_code
+		update_site_meta(
+			$book_id,
+			self::LICENSE_CODE,
+			$supported_types[ $metadata['pb_book_license'] ?? 'all-rights-reserved' ]['abbreviation'] ?? 'All Rights Reserved'
+		);
+
+		// pb_license_name
+		update_site_meta(
+			$book_id,
+			self::LICENSE_NAME,
+			$supported_types[ $metadata['pb_book_license'] ?? 'all-rights-reserved' ]['desc'] ?? 'all-rights-reserved'
+		);
```

Comment:
I'd prefer to extract the license to a variable here so it's easier to read.

```suggestion
		$book_license = $metadata['pb_book_license'] ?? 'all-rights-reserved';

		update_site_meta(
			$book_id,
			self::LICENSE_CODE,
			$supported_types[ $book_license ]['abbreviation'] ?? 'All Rights Reserved'
		);

		// pb_license_name
		update_site_meta(
			$book_id,
			self::LICENSE_NAME,
			$supported_types[ $book_license ]['desc'] ?? 'all-rights-reserved'
		);
```